wicked_pdf.gemspec

```diff
@@ -21,7 +21,7 @@ DESC
     'changelog_uri' => 'https://github.com/mileszs/wicked_pdf/blob/master/CHANGELOG.md'
   }
 
-  spec.required_ruby_version = Gem::Requirement.new('>= 2.2')
+  spec.required_ruby_version = Gem::Requirement.new('>= 2.6')
```

Comment:
Is there a reason to raise this minimum Ruby version, and by so much? I'd like people on older rubies to still be able to use this gem.

Reply:
We can maintain this old ruby version, if you prefer, no problem 🤝 But CI tests may not guarantee it works well on such older versions of the gem

Reply:
I made rollback on this change, returning to ruby 2.2 👍

Reply:
My two cents here, I usually keep CI to run for old versions as long as tests pass.
Once we hit a failure, then we take the decision if it's worth fixing or moving on, deprecating the failing version.